src/components/pages/login-email-code.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable camelcase */
+import React from "react"
+
+import { config, translate, history, useAuthContext, ajax } from "store/index"
+
+import Link from "components/link"
+import Icon from "components/icon"
+import { Spinner } from "@galoymoney/react"
+import { useEmailQuery } from "graphql/generated"
+
+type FCT = React.FC
+
+const LoginEmailCode: FCT = () => {
+  const { setAuthSession } = useAuthContext()
+  const [emailAddress, setEmailAddress] = React.useState("")
+  const [emailLoginId, setEmailLoginId] = React.useState("")
+  const [loading, setLoading] = React.useState(false)
+  const [errorMessage, setErrorMessage] = React.useState<string>("")
+  const [step, setStep] = React.useState<"enterEmail" | "enterCode" | "emailComplete">(
+    "enterEmail",
+  )
+  const emailQuery = useEmailQuery({
+    skip: !emailAddress,
+  })
+
+  const submit: React.FormEventHandler<HTMLFormElement> = async (event) => {
+    event.preventDefault()
+    setErrorMessage("")
+    switch (step) {
+      case "enterEmail": {
+        const emailInput = event.currentTarget.email.value
+        setEmailAddress(emailInput)
+        setLoading(true)
+        const res = await ajax.post(config.galoyAuthEndpoint + "/email/code", {
+          email: emailInput,
+        })
+        if (res.error) {
+          setErrorMessage(res.error)
+          return
+        }
+        setEmailLoginId(res.result)
+        setStep("enterCode")
+        setLoading(false)
+        break
+      }
+      case "enterCode": {
+        const code = event.currentTarget.code.value
+        setLoading(true)
+        const session = await ajax.post(
+          config.galoyAuthEndpoint + "/email/login/cookie",
+          {
+            emailLoginId,
+            code,
+          },
+        )
+        setLoading(false)
+
+        if (session?.error) {
+          setErrorMessage(session.error)
+          return
+        }
+
+        const hasKratosId = Boolean(session.identity.kratosUserId)
+        const isVerified = Boolean(emailQuery.data?.me?.email?.verified)
+        if (!isVerified) {
+          setErrorMessage(
+            "Email is not verified. Please log back in with phone number and goto settings and verify your email address",
+          )
+          return
+        }
```

Comment:
I don't think this block is useful. I think it will verify the email is it's not verify on login (currently at least). this is a TODO on the backend to change the behavior